drivers/samsung/displays/mdc_protocol_spec.cr

```diff
@@ -0,0 +1,55 @@
+  # [header, command, id, data.size, [data], checksum]
+require "placeos-driver/interface/muteable"
```

Comment:
This require shouldn't be needed here.

Reply:
Sorry that was remnants of trying to test the mute function by passing MuteLayer in the spec